changelog/fragments/1734959878-Replace-default-Ubuntu-based-images-with-UBI-minimal-based-ones.yaml

```diff
@@ -0,0 +1,32 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: feature
```

Comment:
Don't we want to mark it as a breaking change? While technically this probably is not a breaking change,  changing the base image from Ubuntu (based on Debian and deb packages) to UBI (based on Red Hat and rpm packages) is quite significant.

Reply:
thanks @andrzej-stencel this is more than a valid point! Evidently when I did the initial "assessment" in my head I perceived this as a more lightweight change, but reading again your point I agree 100%. I just pushed a commit marking this PR as a breaking change 🙂